### DISCUSSION TITLE
Bulk import files

### DISCUSSION THREAD

**Discussion by Nomad-Nathan** [AntlerVC/firetable#1006] [Idea / Feature Request]
It would be really usefull to bulk import files, by downloading the file from an url in the csv.

**Comment by BeeBombshell** [comment]
Hey @Nomad-Nathan, I'm not very sure what you mean by "downloading the file from an url in the csv". However, it is possible to bulk import and update fields in a Rowy table by using the `import by csv` option. Refer to [this](https://github.com/rowyio/rowy/discussions/887) discussion for more insight on the topic. 
If this is not what you're looking for, kindly explain more so that we can discuss this further.
